install

```diff
@@ -33,9 +33,8 @@ if [[ "$OSTYPE" == "msys" ]]; then
 fi
 
 # sudo-check
-if [ $EUID -ne 0 ] && [ "$ON_WINDOWS" == false ]; then
-	_goto_install_error 'Please run this script in sudo mode or as root user'
-	exit
+iif [ $EUID -ne 0 ] && [ "$ON_WINDOWS" == false ]; then
```

Comment:
Can you please check if your change is working? Is this a typo?

Reply:
Yes it is a typo. Whoops

Reply:
You can change that to **if**

Reply:
Feel free to change it